receiver/elasticapmreceiver/receiver.go

```diff
@@ -300,6 +276,96 @@ func (r *elasticAPMReceiver) processBatch(ctx context.Context, batch *modelpb.Ba
 	return errors.Join(errs...)
 }
 
+func (r *elasticAPMReceiver) elasticMetricsToOtelMetrics(rm *pmetric.ResourceMetrics, event *modelpb.APMEvent, timestamp time.Time, ctx context.Context) error {
+	sm := rm.ScopeMetrics().AppendEmpty()
+	metricset := event.GetMetricset()
+
+	// span_breakdown metrics don't have Samples - value is stored directly in event.Span.SelfTime.*
+	if metricset.Name == "span_breakdown" {
+		r.translateBreakdownMetricsToOtel(rm, event)
+		return nil
+	}
+
+	samples := metricset.GetSamples()
+
+	// TODO interval, doc_count
```

Comment:
These TODOs were originally in the code and just moved them. Not sure we need to cover those.